grails-app/domain/org/pih/warehouse/inventory/CycleCountItem.groovy

```diff
@@ -56,6 +56,7 @@ class CycleCountItem {
                 status: status,
                 quantityOnHand: quantityOnHand,
                 quantityCounted: quantityCounted,
+                discrepancy: discrepancy,
```

Comment:
Let's call this quantityVariance.

Reply:
It was expected to be named `discrepancy` in the docs. Have you guys had a conversation around that, or it's just your loose thought to rename that while giving the review?